package.json

```diff
@@ -6,7 +6,7 @@
     "@bugsnag/js": "^7.5.1",
     "@bugsnag/plugin-react": "^7.3.5",
     "@fortawesome/fontawesome-svg-core": "^1.2.32",
-    "@fortawesome/free-brands-svg-icons": "^5.15.3",
+    "@fortawesome/free-brands-svg-icons": "^5.15.1",
```

Comment:
You changed the version here, yet didn't run `npm install` to update the `package-lock.json`, which means for all intents and purposes, it's not changed/fixed.